### PR TITLE
Added creation of a log file for the UI tests

### PR DESCRIPTION
**Pull request by NogaTal98**
- At the end of running each UI test, a log file will be created in /logs/ui_logs
- Add logs to submariner add on cypress test

### CODE REVIEW

File: cypress/cypress/support/e2e.js [hunk 1]
```diff
@@ -15,3 +15,19 @@
...
+    if (log.end) {
+        logs[num] = log.message
+        num++
+    }
+})
+
+let name = Cypress.spec.name
+
+after(() => {
+  cy.writeFile('../logs/ui_logs/'+name+'.log.json', logs)
```
Comment (MaxBab) [inline cypress/cypress/support/e2e.js:32]: Thanks for adding it.

We are storing our tests logs in the following path:
logs/submariner/tests_logs/e2e/ - for api tests.
logs/submariner/tests_logs/ui/ - for ui tests.

Here, we are defining the path to the ui logs:
https://github.com/stolostron/acmqe-mcn-test/blob/main/variables#L13

Could you use this variable to provide the path to the logs for ui?

This way we are controlling the path for the logs from a single place.

Comment (NogaTal98) [reply cypress/cypress/support/e2e.js:32]: Sure, no problem.
Is the type of the file `.log.json` ok? Or should it be `.yaml`?

Comment (MaxBab) [reply cypress/cypress/support/e2e.js:32]: How is the file looks like?
If it's a simple log file, you could leave it as ...log.

Or does it have json structure?

Comment (NogaTal98) [reply cypress/cypress/support/e2e.js:32]: No it does not have json structure.
I will leave it as log.

**Review by MaxBab — APPROVED** [review]

**Pull request merged**